apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ResultStream.cs

```diff
@@ -50,7 +51,7 @@ internal sealed class ResultStream : IAsyncStreamReader<PartialResultSet>, IDisp
         private readonly LinkedList<PartialResultSet> _buffer;
         private readonly SpannerClient _client;
         private readonly ReadOrQueryRequest _request;
-        private readonly Session _session;
+        private readonly PooledSession _session;
```

Comment:
Reason for this change: Because we need to invoke ExecuteWithTransactionSelector from PooledSession Class.